Program.cs

```diff
@@ -1,7 +1,72 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿// --------------------------------------------------------------------------------------------
+// Training ~ A training program for new interns at Metamation, Batch - July 2023
+// Copyright (c) Metamation India.                                              
+// ------------------------------------------------------------------------
+// Implement a Queue<T> using arrays as the underlying data structure. The queue should grow the array when needed (like the TStack above does).
+// If the array does not have to be grown, both Enqueue and Dequeue should be constant time (O(1)) operations.Throw exceptions as needed. 
+// class TQueue<T> {
+//   public void Enqueue (T a) { }
+//   public T Dequeue () { }
+//   public T Peek () { }
+//   public bool IsEmpty { get; }
+// }
+// --------------------------------------------------------------------------------------------
+namespace Training {
+   internal class Program {
+      static void Main () { }
+   }
+   #region TQueue ------------------------------------------------------------------------------
+   /// <summary>TQueue</summary>
+   /// <typeparam name="T"></typeparam>
+   public class TQueue<T> {
+      #region Properties --------------------------------------------
+      /// <summary>Shows count of the Queue</summary>
+      public int Count => mCount;
+
+      /// <summary>shows capacity of the Queue</summary>

```

Comment:
```suggestion
      /// <summary>Capacity of the TQueue</summary>

```
All the properties / methods / members should be arranged alphabetically within the regions.